script/gitlabci/job_base.yml

```diff
@@ -47,3 +47,29 @@
   tags:
     - cuda
     - intel
+
+.base_oacc_nvhpc:
+  image: nvcr.io/nvidia/nvhpc:21.7-devel-cuda11.4-ubuntu20.04
+  variables:
+    ALPAKA_CI_UBUNTU_VER: "20.04"
+    # gcc is used to compile the dependencies, NVHPC will be used to compile alpaka
+    CC: gcc
+    CXX: g++
+    CMAKE_C_COMPILER: nvc
+    CMAKE_CXX_COMPILER: nvc++
+    OMP_NUM_THREADS: 2
+    ALPAKA_CI_BUILD_JOBS: 2
+    ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE: "ON"
+    ALPAKA_ACC_CPU_B_SEQ_T_THREADS_ENABLE: "OFF"
+    ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE: "OFF"
+    ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLE: "OFF"
+    ALPAKA_ACC_GPU_CUDA_ENABLE: "OFF"
+    ALPAKA_ACC_CPU_B_SEQ_T_FIBERS_ENABLE: "OFF"
+    ALPAKA_ACC_ANY_BT_OACC_ENABLE: "ON"
+    ALPAKA_OFFLOAD_MAX_BLOCK_SIZE: 256
+    ALPAKA_DEBUG_OFFLOAD_ASSUME_HOST: "OFF"
+    CMAKE_CXX_FLAGS: "-acc -ta=tesla"
+  extends: .base
+  tags:
+    - cuda
+    - intel
```

Comment:
@psychocoderHPC I added this gitlab-ci job. Will this run automatically at HZDR fron this PR, or does it need to be triggered manually? In any case: Where can I see the results?

Reply:
Never mind, found it. Apparently, I did not add the job, just the base. @psychocoderHPC : can you maybe kill the previous CI job I produced with wrong configs?

Reply:
I killed the GitLab CI pipeline.

Reply:
Thanks, but that was the latest one. This one should run. There are some skipped ones...

Reply:
Thanks. Can you please restart the latest one? That one should run...

Reply:
I restarted the last one and the other was canceled by anybody else.

Reply:
You can push any time new commits or force push to your PR. The HZDR CI will automatically kill old instances of the same PR.